src/ArmTemplates/Common/API/Clients/Abstractions/ApiClientBase.cs

```diff
@@ -30,11 +31,18 @@ public ApiClientBase(IHttpClientFactory httpClientFactory)
             this.httpClient = httpClientFactory.CreateClient();
         }
 
-        protected async Task<string> CallApiManagementAsync(string azToken, string requestUrl, bool useCache = true, ClientHttpMethod method = ClientHttpMethod.GET)
+        /// <summary>
+        ///  Main method to call trigger api management emdpoint.
+        /// </summary>
+        /// <returns>
+        /// string containing response body
+        /// bool indicating if empty default response of the requested resource should be returned
+        /// </returns>
+        protected async Task<(string,bool)> CallApiManagementAsync(string azToken, string requestUrl, bool useCache = true, ClientHttpMethod method = ClientHttpMethod.GET, bool catchMethodNotAllowed = false)
```

Comment:
I still dont get what is the point of `bool` here.

Reply:
will change the approach since there is a possibility to fetch sku, thanks